### PR TITLE
#ORCOMP-216 Extracted all resources in CS into resx files

### PR DESCRIPTION
**Pull request by alexfdezsauco**
Fixes #ORCOMP-216.

### Changes proposed in this pull request:

- Extracted all resources in cs into resx files

### CODE REVIEW

File: src/Orc.FilterBuilder/Orc.FilterBuilder.Shared.WPF/ViewModels/FilterBuilderViewModel.cs [hunk 1]
```diff
@@ -235,7 +238,7 @@ private bool OnDeleteSchemeCanExecute(FilterScheme filterScheme)
         private async void OnDeleteSchemeExecute(FilterScheme filterScheme)
 #pragma warning restore AvoidAsyncVoid
         {
-            if (await _messageService.ShowAsync($"Are you sure you want to delete filter '{filterScheme.Title}'?", "Delete filter?", MessageButton.YesNo) == MessageResult.Yes)
+            if (await _messageService.ShowAsync(string.Format(_languageService.GetString("FilterBuilder_ShowAsync_Message_AreYouSureYouWantToDeleteFilter0Question_Pattern"), filterScheme.Title), _languageService.GetString("FilterBuilder_ShowAsync_DeleteFilterQuestion_Caption"), MessageButton.YesNo) == MessageResult.Yes)
```
Comment (GeertvanHorrik) [inline src/Orc.FilterBuilder/Orc.FilterBuilder.Shared.WPF/ViewModels/FilterBuilderViewModel.cs]: What is the 0 doing in "DeleteFilter0Question"?

File: src/Orc.FilterBuilder/Orc.FilterBuilder.Shared/Expressions/BooleanExpression.cs [hunk 2]
```diff
@@ -43,7 +45,7 @@ public override bool CalculateResult(IPropertyMetadata propertyMetadata, object
                     return entityValue == Value;
 
                 default:
-                    throw new NotSupportedException(string.Format("Condition '{0}' is not supported.", SelectedCondition));
+                    throw new NotSupportedException(string.Format(this.GetDependencyResolver().Resolve<ILanguageService>().GetString("FilterBuilder_Exception_Message_Condition0IsNotSupported_Pattern"), SelectedCondition));
```
Comment (GeertvanHorrik) [inline src/Orc.FilterBuilder/Orc.FilterBuilder.Shared/Expressions/BooleanExpression.cs]: Use LanguageHelper instead

File: src/Orc.FilterBuilder/Orc.FilterBuilder.Shared.WPF/Converters/ConditionTreeItemConverter.cs [hunk 3]
```diff
@@ -25,7 +27,7 @@ protected override object Convert(object value, Type targetType, object paramete
                     return value is PropertyExpression ? Visibility.Visible : Visibility.Collapsed;
 
                 default:
-                    throw new NotSupportedException(string.Format("Parameter {0} is not supported.", parameter));
+                    throw new NotSupportedException(string.Format(this.GetDependencyResolver().Resolve<ILanguageService>().GetString("FilterBuilder_Exception_Message_Parameter0IsNotSupported_Pattern"), parameter));
```
Comment (GeertvanHorrik) [inline src/Orc.FilterBuilder/Orc.FilterBuilder.Shared.WPF/Converters/ConditionTreeItemConverter.cs]: Use LanguageHelper instead

File: src/Orc.FilterBuilder/Orc.FilterBuilder.Shared/Expressions/TimeSpanExpression.cs [hunk 4]
```diff
@@ -87,7 +89,7 @@ protected override void OnPropertyChanged(AdvancedPropertyChangedEventArgs e)
                         break;
 
                     default:
-                        throw new NotSupportedException(string.Format("Type '{0}' is not supported.", SelectedSpanType));
+                        throw new NotSupportedException(string.Format(this.GetDependencyResolver().Resolve<ILanguageService>().GetString("FilterBuilder_Exception_Message_Type0IsNotSupported_Pattern"), SelectedSpanType));
```
Comment (GeertvanHorrik) [inline src/Orc.FilterBuilder/Orc.FilterBuilder.Shared/Expressions/TimeSpanExpression.cs]: Use LanguageHelper instead

File: src/Orc.FilterBuilder/Orc.FilterBuilder.Shared/Expressions/TimeSpanExpression.cs [hunk 5]
```diff
@@ -116,7 +118,7 @@ public override bool CalculateResult(IPropertyMetadata propertyMetadata, object
                     return entityValue <= Value;
 
                 default:
-                    throw new NotSupportedException(string.Format("Condition '{0}' is not supported.", SelectedCondition));
+                    throw new NotSupportedException(string.Format(this.GetDependencyResolver().Resolve<ILanguageService>().GetString("FilterBuilder_Exception_Message_Condition0IsNotSupported_Pattern"), SelectedCondition));
```
Comment (GeertvanHorrik) [inline src/Orc.FilterBuilder/Orc.FilterBuilder.Shared/Expressions/TimeSpanExpression.cs]: Use LanguageHelper instead

File: src/Orc.FilterBuilder/Orc.FilterBuilder.Shared/Expressions/ValueDataTypeExpression.cs [hunk 6]
```diff
@@ -65,7 +66,7 @@ public override bool CalculateResult(IPropertyMetadata propertyMetadata, object
                         return entityValue != null;
 
                     default:
-                        throw new NotSupportedException(string.Format("Condition '{0}' is not supported.", SelectedCondition));
+                        throw new NotSupportedException(string.Format(this.GetDependencyResolver().Resolve<ILanguageService>().GetString("FilterBuilder_Exception_Message_Condition0IsNotSupported_Pattern"), SelectedCondition));
```
Comment (GeertvanHorrik) [inline src/Orc.FilterBuilder/Orc.FilterBuilder.Shared/Expressions/ValueDataTypeExpression.cs]: Use LanguageHelper instead

File: src/Orc.FilterBuilder/Orc.FilterBuilder.Shared/Expressions/ValueDataTypeExpression.cs [hunk 7]
```diff
@@ -92,7 +93,7 @@ public override bool CalculateResult(IPropertyMetadata propertyMetadata, object
                         return _comparer.Compare(entityValue, Value) <= 0;
 
                     default:
-                        throw new NotSupportedException(string.Format("Condition '{0}' is not supported.", SelectedCondition));
+                        throw new NotSupportedException(string.Format(this.GetDependencyResolver().Resolve<ILanguageService>().GetString("FilterBuilder_Exception_Message_Condition0IsNotSupported_Pattern"), SelectedCondition));
```
Comment (GeertvanHorrik) [inline src/Orc.FilterBuilder/Orc.FilterBuilder.Shared/Expressions/ValueDataTypeExpression.cs]: Use LanguageHelper instead

**Comment by alexfdezsauco** [comment]
Fixed

**Pull request merged**